src/EnergyPlus/DataPhotovoltaics.hh

```diff
@@ -60,6 +60,8 @@ namespace EnergyPlus {
 
 namespace DataPhotovoltaics {
 
+    Real64 constexpr MinIrradiance = 0.3; // [W/m2] Assume no operation if Ic is below this number (W/m2)
+
     enum class PVModel
     {
         Unassigned,
```

Comment:
`Unassigned` needs to be explicitly assigned to -1, otherwise it will be assigned to 0 by default.